dev-tools/test_cov.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# This script executes the tests and starts the database docker container if necessary.
+
+# Check if local postgres server is running
+if nc -w1 localhost 5432; then
+
+    cd $(dirname "$BASH_SOURCE")/..
+
+    # Execute tests
+    pipenv run integreat-cms-cli test cms
```

Comment:
```suggestion
    pipenv run integreat-cms-cli test cms --set=COVERAGE
```
The coverage parameter is missing if a local postgres database server is used instead of docker.